docs/source/rendering_menus/index.rst

```diff
@@ -7,3 +7,4 @@ Rendering menus
 
     template_tag_reference
     custom_templates
+    use_with_jinja2
```

Comment:
Please remove this for now - We'll work on Jinja-specific stuff as a separate piece of work :)